openedx/features/course_experience/templates/course_experience/course-outline-fragment-new.html

```diff
@@ -14,7 +14,17 @@
 
 <main role="main" class="course-outline-visualprogress" id="main" tabindex="-1">
     % if blocks.get('children'):
-        <ol class="block-tree accordion" role="presentation">
+        <button class="btn btn-primary"
+                id="expand-collapse-outline-all-button"
+                aria-expanded="false"
+                aria-controls="course-outline-block-tree"
+                >
+          <span class="extra-padding-outline" id="expand-collapse-outline-all-span">${_("Expand All")}</span>
```

Comment:
Looking at this again, maybe "expand-collapse-outline-all-extra-padding" ? So it'll be consistent with the other classes and IDs we made in this PR?